run_docker.bat

```diff
@@ -13,19 +13,19 @@ REM WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 REM See the License for the specific language governing permissions and
 REM limitations under the License.
 
-set current_dir=%~dp0
-set default_image=quantconnect/lean:latest
-set default_data_dir=%current_dir%Data\
-set default_results_dir=%current_dir%
-set default_config_file=%current_dir%Launcher\config.json
-set default_python_dir=%current_dir%Algorithm.Python\
-set csharp_dll=%current_dir%Launcher\bin\Debug\QuantConnect.Algorithm.CSharp.dll
-set csharp_pdb=%current_dir%Launcher\bin\Debug\QuantConnect.Algorithm.CSharp.pdb
+set CURRENT_DIR=%~dp0
+set DEFAULT_IMAGE=quantconnect/lean:latest
+set DEFAULT_DATA_DIR=%CURRENT_DIR%Data\
+set DEFAULT_RESULTS_DIR=%CURRENT_DIR%\Results
```

Comment:
`\Results` think `\` isn't required? I'm get `Enter absolute path to store results [default: ...\Lean\\Results]`